src/hooks/useAuthenticated.ts

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from 'react';
+import { USER_SELF } from '@/components/constants/url';
+
+export const DEFAULT_AVATAR = '/Avatar.png';
+
+interface Userdata {
+  userName: string;
+  firstName: string;
+  profilePicture: string;
+}
+
+interface HooksReturnType {
+  userData: Userdata;
+  isLoggedIn: boolean;
+  isLoading: boolean;
+}
+
+const useAuthenticated = (): HooksReturnType => {
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  const [userData, setUserData] = useState<Userdata>({
+    userName: '',
+    firstName: '',
+    profilePicture: '',
+  });
+
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true);
+      await fetch(USER_SELF, { credentials: 'include' })
+        .then((response) => {
+          if (!response.ok) {
+            setIsLoggedIn(false);
+            console.log('response :>> ', response);
```

Comment:
!

Reply:
> Could you attach screenshots of how the UI looks in mobile view?

![image](https://user-images.githubusercontent.com/42349041/191519352-2bd42414-0ee5-4fbf-af5b-6ab8633632c5.png)
@akshay1502 the navbar on status site is not responsive yet. I have created an issue for it here https://github.com/Real-Dev-Squad/website-status/issues/317

Reply:
done